datastore/integration_test.go

```diff
@@ -161,13 +183,27 @@ func newClient(ctx context.Context, t *testing.T, dialOpts []grpc.DialOption) *C
 	for _, opt := range dialOpts {
 		opts = append(opts, option.WithGRPCDialOption(opt))
 	}
-	client, err := NewClient(ctx, testutil.ProjID(), opts...)
+	client, err := NewClientWithDatabase(ctx, testutil.ProjID(), testParams["databaseID"].(string), opts...)
 	if err != nil {
-		t.Fatalf("NewClient: %v", err)
+		t.Fatalf("NewClientWithDatabase: %v", err)
 	}
 	return client
 }
 
+func TestIntegration_NewClient(t *testing.T) {
+	if testing.Short() {
```

Comment:
just for my understanding, what is short mode? When is it run vs the full suite?

Reply:
Short mode runs only unit tests. Kokoro uses short mode pre submit: https://github.com/googleapis/google-cloud-go/blob/main/internal/kokoro/presubmit.sh#L57